packages/headless/src/controllers/commerce/core/facets/generator/headless-commerce-facet-generator.ts

```diff
@@ -156,6 +165,11 @@ export function buildFacetGenerator(
   return {
     ...controller,
 
+    deselectAll: () => {
+      dispatch(deselectAllBreadcrumbs());
```

Comment:
You'll have to make some adjustments when you sync your branch with master.

The action you should dispatch is `clearAllCoreFacets`, which is defined in `features/commerce/facets/core-facet/core-facet-actions.ts`

Reply:
Perfect ty!